make/photon/db/Dockerfile.base

```diff
@@ -6,9 +6,16 @@ RUN tdnf install -y shadow >> /dev/null \
     && groupadd -r postgres --gid=999 \
     && useradd -m -r -g postgres --uid=999 postgres
 
-RUN tdnf install -y postgresql14-server >> /dev/null
-RUN tdnf install -y gzip postgresql15-server findutils bc >> /dev/null \
-    && mkdir -p /docker-entrypoint-initdb.d \
+# Detect architecture and install different packages accordingly
+ARG TARGETARCH
+RUN if [ "$TARGETARCH" = "amd64" ]; then \
```

Comment:
Why do we install sql14 for AMD64 but not for ARM?

Reply:
As per my comment, I think that is unnecessary.

I'm not a harbor maintainer, so you shouldn't listen to me, but I think it can still be left out (depending on whether the maintainers think it's necessary).